CI/doc/source/conf.py

```diff
@@ -0,0 +1,146 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import contextlib
+import os
+import shutil
+import sys
+from typing import List
+
+sys.path.insert(0, os.path.abspath("../.."))
+sys.setrecursionlimit(1500)
+
+# Move logos over to doc directory
+p = os.path.join("_static", "logos")
+if not os.path.exists(p):
+    os.mkdir("_static/logos")
+
+for filename in os.listdir(os.path.join("..", "..", "..", "logos")):
+    if filename.endswith(".png"):
+        shutil.copy(
+            os.path.join("..", "..", "..", "logos", filename),
+            os.path.join("_static", "logos", filename),
+        )
+        fn = os.path.join("_static", "logos", filename)
+        from PIL import Image
+
+        im = Image.open(fn)
+        # Remove left 30% of image
+        #im = im.crop((int(im.size[0] * 0.45), 0, int(im.size[0] * 1), im.size[1]))
+        im = im.crop((int(im.size[0] * 0.32), 0, int(im.size[0] * 1), im.size[1]))
+        im.save(fn.replace(".png", "_cropped.png"))
+
+
+# -- Project information -----------------------------------------------------
+
+project = "Analog Devices, Inc. Transceiver Toolbox"
+copyright = "2019-2022, Analog Devices, Inc"
+author = "Analog Devices, Inc."
+
+# The full version, including alpha/beta/rc tags
+release = "v22.2.1"

```

Comment:
This should probably reflect what its tested against

Reply:
Fixed.